transcription/email.go

```diff
@@ -0,0 +1,30 @@
+package transcription
+
+import (
+	"log"
+	"net/smtp"
+	"os"
+	"strings"
+)
+
+func sendEmail(from string, to []string, subject string, body string) {
+	// Set up authentication information.
+	auth := smtp.PlainAuth(
+		"",
+		os.Getenv("MAIL_EMAIL"),
+		os.Getenv("MAIL_PASSWORD"),
+		"smtp.gmail.com",
+	)
+
+	// Connect to the server, authenticate, set the sender and recipient,
+	// and send the email all in one step.
+	msg := []byte("To: " + strings.Join(to, ", ") + "\r\n" +
```

Comment:
Why do you need to define `msg`? Shouldn't the email message be passed as an argument?